src/esp/bindings/PhysicsBindings.cpp

```diff
@@ -20,7 +20,8 @@ void initPhysicsBindings(py::module& m) {
       .value("UNDEFINED", MotionType::UNDEFINED)
       .value("STATIC", MotionType::STATIC)
       .value("KINEMATIC", MotionType::KINEMATIC)
-      .value("DYNAMIC", MotionType::DYNAMIC);
+      .value("DYNAMIC", MotionType::DYNAMIC)
+      .value("VISUALIZATION_ONLY", MotionType::VISUALIZATION_ONLY);
```

Comment:
Can we rename MotionType? It feels like we're abusing this now. VISUALIZATION_ONLY doesn't really refer to how this thing moves. It moves exactly the same as KINEMATIC, true?

Reply:
True, we could re-name the type to KINEMATIC_VISUALIZATION_ONLY or we could re-name MotionType to something else. Thoughts?

Reply:
@aclegg3  Actually, wouldn't visualization only be the same an `MotionType::DISABLED` After all, nothing can be move it ? I would prefer `MotionType::ETHEREAL`.